src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -1558,7 +1567,7 @@ abstract class RefChecks extends Transform {
         // analyses in the pattern matcher
         if (!inPattern) {
           checkImplicitViewOptionApply(tree.pos, fn, args)
-          checkSensible(tree.pos, fn, args)
+          checkSensible(tree.pos, fn, args) // TODO: this should move to preEraseApply, as reasoning about runtime semantics makes more sense in the JVM type system
```

Comment:
I realized that on my daughter's macbook, I can 2-finger scroll right. Previously, I would select and scroll on long adriaaaaan lines. You never know how many a's.